src/Stryker.ts

```diff
@@ -92,6 +93,17 @@ export default class Stryker {
     });
   }
 
+  private loadDefaultPlugins() {
```

Comment:
Maybe we should not call this 'default plugins'. Maybe we could better call it `loadDefaultMutators` and add it as a method to the `MutatorOrchestrator`. I don't see a single mutator as being a plugin. So also don't use the `PluginLoader`. That way, a mutators is no longer responsible by itself to add it to the `MutatorFactory` (that should generaly only be done by a plugin)